skimage/morphology/greyreconstruct.py

```diff
@@ -42,15 +42,20 @@ def reconstruction(seed, mask, method='dilation', selem=None, offset=None):
         are dilated or eroded.
     mask : ndarray
         The maximum (dilation) / minimum (erosion) allowed value at each pixel.
-    method : {'dilation'|'erosion'}
+    method : {'dilation'|'erosion'}, optional
         Perform reconstruction by dilation or erosion. In dilation (or
         erosion), the seed image is dilated (or eroded) until limited by the
         mask image. For dilation, each seed value must be less than or equal
         to the corresponding mask value; for erosion, the reverse is true.
-    selem : ndarray
+        Default is 'dilation'.
```

Comment:
Typically we don't include the defaults here when they are available from the function signature...